apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -2,7 +2,18 @@
 /**
  * External dependencies
  */
-import { find, isEmpty, reduce, get, keyBy, mapValues, memoize, filter, sortBy } from 'lodash';
+import {
+	find,
+	isEmpty,
+	reduce,
+	get,
+	keyBy,
+	mapValues,
+	memoize,
+	filter,
+	sortBy,
+	stubTrue,
+} from 'lodash';
```

Comment:
These reformatting changes make it difficult to see what was added. However, given it was just `stubTrue` that's ok 👍 😄

Reply:
only `stubTrue` ✅